src/Runner/Test.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * This file is part of the Nette Tester.
+ * Copyright (c) 2009 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Tester\Runner;
+
+
+/**
+ * Test represents one result.
+ */
+class Test
+{
+	const
+		NO_RESULT = 0, # TODO: lepsi jmeno
+		PASSED = 1,
+		SKIPPED = 2,
+		FAILED = 3;
+
+	/** @var string|NULL */
+	public $title;
+
+	/** @var string */
+	public $message;
+
+	/** @var string|NULL */
+	public $stdout;
+
+	/** @var string|NULL */
+	public $stderr;
+
+	/** @var string */
+	private $file;
+
+	/** @var int */
+	private $result = self::NO_RESULT;
+
+	/** @var string[][] */
+	private $args = [];
+
+
+	/**
+	 * @param  string
+	 * @param  string
+	 */
+	public function __construct($file, $title = NULL)
+	{
+		$this->file = $file;
+		$this->title = $title;
+	}
+
+
+	/**
+	 * @return string
+	 */
+	public function getFile()
+	{
+		return $this->file;
+	}
+
+
+	/**
+	 * @return string[][]
+	 */
+	public function getArguments()
+	{
+		return $this->args;
+	}
+
+
+	/**
+	 * @return array
+	 */
+	public function getJobArguments()  # TODO: tohle sem asi nepatri
```

Comment:
Now that the arguments are abstracted in an array, maybe `Job` should just call `getArguments()` and do this formatting itself? It feels coupled to the way Jobs are executed.